php/WP_CLI/CommandWithUpgrade.php

```diff
@@ -264,18 +264,24 @@ protected function update_many( $args, $assoc_args ) {
 			\WP_CLI::error( $line );
 		}
 
-		if ($num_to_update > 0) {
-			$status = array();
-			foreach($items_to_update as $item_to_update => $info) {
-				$status[$item_to_update] = array(
-					'name' => $info['name'],
-					'old_version' => $info['version'],
-					'new_version' => $info['update_version'],
-					'status' => $result[$item_to_update] !== null ? 'Updated' : 'Error',
-				);
+		if ( $num_to_update > 0 ) {
+			if ( ! empty( $assoc_args['format'] ) && 'summary' === $assoc_args['format'] ) {
+				foreach( $items_to_update as $item_to_update => $info ) {
+					$message = $result[$item_to_update] !== null ? 'updated successfully' : 'did not update';
```

Comment:
PHP Notice:  Undefined index: /var/www/subdirwp/server/static/themes/decode in php/WP_CLI/CommandWithUpgrade.php on line 270